JS-files/spk.js

```diff
@@ -0,0 +1,53 @@
+    Header: 'Dr.Ernest-Darkoh',
+    introInfo: 'Expert on health systems strengthening and large-scale program implementation.',
+    detail: ' He is on the Board of the Schwab Foundation for Social Entrepreneurship and has been recognized by New African Magazine as one of the 100 most influential Africans and named one of eighteen “Global Health Heroes” by TIME Magazine.',
+  },
+  {
+    imageLink: './Images/Dr.-Yvonne-Mburu.jpg',
+    Header: 'Dr.Yvonne-Mburu',
+    introInfo: 'Founder of Nexakili, a growing network of African health professionals and scientists.',
+    detail: 'Her academic research at the Curie Institute in Paris and at the University of Pittsburgh, PA originally focused on the immunology and immunotherapy of cancer. Her work also spans health systems and policies, with a key focus on integrating breakthrough technologies to accelerate the pace of innovative medicine especially in Africa.',
+  },
+  {
+    imageLink: './Images/Professor-Tom-Kariuki-1532x1536.jpg',
+    Header: 'Professor-Tom-Kariuki',
+    introInfo: 'Prof Thomas Kariuki is Executive Director of the Science for Africa (SFA) Foundation.',
+    detail: 'Dr Kariuki previously served as the Director of Programmes for the Alliance for Accelerating Excellence in Science in Africa (AESA) and Director of a Biomedical facility, the Institute of Primate Research / National Museums of Kenya where he established his infectious immunology research group.',
+  },
+  {
+    imageLink: './Images/Professor-Helen-Rees-.jpg',
+    Header: 'Professor-Helen-Rees',
+    introInfo: 'Professor Helen Rees is the Founder and Executive Director of Wits RHI, the largest research Institute at Wits University.',
+    detail: 'Professor Rees chairs the Board of the South African Health Products Regulatory Authority and also Chairs the World Health Organisation’s AFRO Region Immunization Technical Advisory Group. Also a member of WHO’s Scientific and Technical Advisory Group on Infectious Hazards. ',
+  },
+  {
+    imageLink: './Images/Dr.-Vera-Songwe..jpg',
+    Header: 'Dr.-Yvonne-Mburu',
+    introInfo: 'Dr. Vera Songwe is the Under-Secretary-General of the United Nations and Executive Secretary of the Economic Commission for Africa (ECA).',
+    detail: 'Upon her appointment, she became the first woman to lead the institution in its 60-year history. She was listed as one of Africa’s 50 most powerful women by Forbes in 2020 and named as one of the ‘100 Most Influential Africans’ by Jeune Afrique in 2019. In 2017.',
+  },
+];
+
+thirdSection.innerHTML = '';
+for (let i = 0; i < FeatureArray.length; i += 1) {
+  thirdSection.innerHTML = `${thirdSection.innerHTML}
+   <div class= "speaker-detail"> <img src="${FeatureArray[i].imageLink}">
+     <div class="speakers">
+        <h3>${FeatureArray[i].Header}</h3>
+        <small>${FeatureArray[i].introInfo}</small>
+        <hr>
+        <p>${FeatureArray[i].detail}</p>
+      </div> 
+    <div>`;
+}
```

Comment:
- [ ] [OPTIONAL] Prefer to use ES6 higher-order functions in situations like this. it helps avoid unnecessary bugs and is much more concise and readable. 👇

```suggestion
FeatureArray.map((speaker) => {
  thirdSection.innerHTML = `${thirdSection.innerHTML}
   <div class= "speaker-detail"> <img src="${speaker.imageLink}">
     <div class="speakers">
        <h3>${speaker.Header}</h3>
        <small>${speaker.introInfo}</small>
        <hr>
        <p>${speaker.detail}</p>
      </div> 
    <div>`;
});
```

Reply:
Hi @MrBrN197. As requested by you, I have;
1. Highlighted a section of my code that I wrote in my video.
2. Deployed my project properly and provided a correct link,
3.  Ensured that the image for the speaker is perfectly square and is not squeezed when resizing,
4. Added the checkboard pattern behind the featured speakers' image,
5.  Used the correct font family (Lato),
6.  Created some padding around the container(main program section), so it matches the reference closer,
7.  Aligned horizontally the images(in the conference section) in the desktop version,
8.  Made the about page in the desktop version to have a dark background,
9. on the mobile version of this card,  reduced the gap between each gap,
10. Ensured that every img tag has an appropriate alt text.